sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1641,6 +1639,7 @@ def __init__(
       ignore_insert_ids,
       ignore_unknown_columns,
       with_auto_sharding,
+      num_streaming_keys,
```

Comment:
Is there a reason you don't have the old default value set here? It would help mitigate issues with backwards compatibility for users who aren't using the `test_client` and `max_retries` args either

Reply:
Done. Thanks!